cmd/web.go

```diff
@@ -203,6 +203,10 @@ func runWeb(ctx *cli.Context) error {
 		m.Post("/sign_up", bindIgnErr(auth.RegisterForm{}), user.SignUpPost)
 		m.Get("/reset_password", user.ResetPasswd)
 		m.Post("/reset_password", user.ResetPasswdPost)
+		m.Get("/2fa", user.ShowTwofa)
+		m.Post("/2fa", bindIgnErr(auth.TwofaAuthForm{}), user.TwofaPost)
+		m.Get("/2fa_scratch", user.TwofaScratch)
+		m.Post("/2fa_scratch", bindIgnErr(auth.TwofaScratchAuthForm{}), user.TwofaScratchPost)
```

Comment:
Since there are many endpoints it might be better to use the following structure and change 2fa_scratch to be 2fa/scratch.

    m.Group("/2fa", func() {
        m.Get("", user.ShowTwofa)
        [...]
    })